Source/Lib/Codec/EbEncHandle.c

```diff
@@ -552,10 +552,31 @@ EB_U64 GetAffinityMask(EB_U32 lpnum) {
 }
 #endif
 
+void SwitchToRealTime()
+{
+
+#if  __linux__
+
+    struct sched_param schedParam = {
+        .sched_priority = 99
```

Comment:
use sched_get_priority_max(SCHED_FIFO) instead of 99?

Reply:
That would freeze the machine :)

Reply:
I thought sched_get_priority_max(SCHED_FIFO) is 99?

Reply:
set to sched_get_priority_max(SCHED_FIFO)